README.md

```diff
@@ -4,6 +4,10 @@
 
 ``ImSwitch`` is a software solution in Python that aims at generalizing microscope control by using an architecture based on the model-view-presenter (MVP) to provide a solution for flexible control of multiple microscope modalities.
 
+## News 
+
+🎥 Watch this intro video from Jacopo about ImSwitch on [YouTube](https://www.youtube.com/watch?v=B54QCt5OQPI&ab_channel=Euro-BioImagingCommunication
```

Comment:
Missing closing bracket

Reply:
@beniroquai this still needs to be resolved

Reply:
ay! Fixed. I would like to keep it there fore now and move it to some other place later (in addition). In case anybody sees this repo first, they can learn from you :)